src/OpenTelemetry.Exporter.Console/ConsoleExporterOptions.cs

```diff
@@ -27,9 +26,9 @@ public class ConsoleExporterOptions
         public ConsoleExporterOutputTargets Targets { get; set; } = ConsoleExporterOutputTargets.Console;
 
         /// <summary>
-        /// Gets or sets the metric export interval in milliseconds. The default value is <c>Timeout.Infinite</c>.
+        /// Gets or sets the metric export interval in milliseconds. The default value is 60000.
         /// </summary>
-        public int MetricExportIntervalMilliseconds { get; set; } = Timeout.Infinite;
+        public int MetricExportIntervalMilliseconds { get; set; } = 60000;
```

Comment:
Why would a console exporter use the periodic exporting strategy?

Reply:
Ah, yes that makes sense now. The side-effect of this change causing the console exporter to default to periodic exporting was not immediately obvious.

https://github.com/open-telemetry/opentelemetry-dotnet/blob/6e6afdbc3614cfc52d20825a25f06bd80b5ba64e/src/OpenTelemetry.Exporter.Console/ConsoleExporterMetricsExtensions.cs#L42-L44

My gut says it would make sense to decouple the idea of periodic vs. non-periodic in a separate config setting. Kinda similar to how tracing has the batch exporter options.

Reply:
Maybe break this into two PRs... because I think the change to the OTLP exporter is correct.

Reply:
Ok. I broke it up.